lib/plugins/Label_21_POS.test.ts

```diff
@@ -0,0 +1,60 @@
+import { MessageDecoder } from '../MessageDecoder';
+import { Label_21_POS } from './Label_21_POS';
+
+describe('Label_21_POS', () => {
+    let plugin: Label_21_POS;
+
+    beforeEach(() => {
+        const decoder = new MessageDecoder();
+        plugin = new Label_21_POS(decoder);
+    });
+
+    test('matches Label 21 Preamble POS qualifiers', () => {
+        expect(plugin.decode).toBeDefined();
+        expect(plugin.name).toBe('label-21-pos');
+        expect(plugin.qualifiers).toBeDefined();
+        expect(plugin.qualifiers()).toEqual({
+          labels: ['21'],
+          preambles: ['POS'],
+        });
+      });
+    
+
+    test('decodes Label 21 Preamble POS valid', () => {
+        const text = 'POSN 39.841W 75.790, 220,184218,17222,22051,  34,- 4,204748,KTPA'
```

Comment:
i usually link to the airframes message, but this is from my own acarshub as i can't search right now